practices/run-pair-programming-sessions.md

```diff
@@ -1,103 +1,129 @@
 # Run Pair Programming Sessions
 
-Run Pair Programming Sessions involve two developers collaborating either at one workstation or remotely, with one typing out code (the driver) and the other reviewing.
-This practice enhances code quality by facilitating constant review, improving knowledge sharing, discussion, and problem-solving.
+In pair programming sessions, two developers collaborate either at one workstation or remotely, with one typing out code (the driver) and the other reviewing (the navigator).
```

Comment:
The driver/navigator is one of multiple ways of pairing. It's one of the more common ways of pairing. Maybe we add the word "usually" before we dive into the driver/navigator bit?

Reply:
That works. So, it would read: "In pair programming sessions, two developers collaborate either at one workstation or remotely. Typically, one types out code (the driver) and the other reviews it (the navigator)." OK?

Reply:
Edited.